src/data/nav/pubsub.ts

```diff
@@ -22,36 +22,9 @@ export default {
       name: 'Getting started',
       pages: [
         {
-          name: 'JavaScript',
-          link: '/docs/getting-started/javascript',
-        },
-        {
-          name: 'React',
-          link: '/docs/getting-started/react',
-        },
-        {
-          name: 'React Native',
-          link: '/docs/getting-started/react-native',
-        },
-        {
-          name: 'Kotlin',
-          link: '/docs/getting-started/kotlin',
-        },
-        {
-          name: 'Swift',
-          link: '/docs/getting-started/swift',
-        },
-        {
-          name: 'Ruby',
-          link: '/docs/getting-started/ruby',
-        },
-        {
-          name: 'Python',
-          link: '/docs/getting-started/python',
-        },
-        {
-          name: 'Quickstart guide',
```

Comment:
Is the intention to remove it from the nav for now, but not delete the page just yet?